server/middlewares/statsGoogleAnalytics.js

```diff
@@ -84,6 +85,13 @@ const trackTxSubmissions = mung.jsonAsync(async (body, req, res) => {
         value: undefined,
       })
 
+      await trackEvent({
+        ...baseEventData,
+        action: txCryptoProviderVersion,
```

Comment:
maybe txWalletVersion would be more accurate of a name. Our crypto providers as such don't really have a version and this may cause the false impression that they do

Reply:
by crypto provider I meant trezor/ledger/mnemonic it self, anyway, renamed to txWalletVersion